doc/radosgw/s3/commons.rst

```diff
@@ -20,9 +20,20 @@ The second method identifies the bucket via a virtual bucket host name. For exam
 
 To configure virtual hosted buckets, you can either set ``rgw_dns_name = cname.domain.com`` in ceph.conf, or add ``cname.domain.com`` to the list of ``hostnames`` in your zonegroup configuration. See `Ceph Object Gateway - Multisite Configuration`_ for more on zonegroups.
 
+.. prompt:: bash $
+
+	ceph config set client.rgw.<ceph authx client for rgw> rgw rgw_dns_name cname.domain.com
+
 .. tip:: We prefer the first method, because the second method requires expensive domain certification and DNS wild cards.
 
-.. tip:: You can define multiple hostname directly with the :confval:`rgw_dns_name` parameter.
+.. tip:: You can define multiple hostnames directly with the :confval:`rgw_dns_name` parameter.
+
+.. tip:: When SSL is enabled, the certificates need to use a wildcard in the domain name so it matches the bucket subdomains.
+
+.. tip:: RGW instances would share a dns name if they're behind the same proxy.Then you can use `ceph config set client.rgw` to set the dns name for all instances. 
```

Comment:
nit: these docs refer to 'RGW' as the 'Ceph Object Gateway'

> .. note:: When Ceph Object Gateways are behind a proxy, the proxy's dns name should be used instead.

Reply:
s/dns/DNS/g